tests/wpt/tests/quirks/line-height-preserved-segment-break.html

```diff
@@ -0,0 +1,27 @@
+<!-- quirks mode -->
+<link rel="author" title="Martin Robinson" href="mrobinson@igalia.com">
+<link rel="help" href="https://quirks.spec.whatwg.org/#the-line-height-calculation-quirk">
+<link rel="help" href="https://drafts.csswg.org/css-text/#valdef-white-space-pre-line">
+<link rel="match" href="line-height-preserved-segment-break-ref.html">
```

Comment:
Maybe use `/css/reference/ref-filled-green-100px-square.xht`? Or do you want the ref to be inside `/quirks/`?